pyincore/spaceservice.py

```diff
@@ -91,6 +109,57 @@ def update_space(self, space_id: str, space_json):
         response = r.json()
         return response
 
+    def add_to_space_by_name(self, space_name, dataset_id):
```

Comment:
Can you add "str" as the types for space_name and dataset_id?

Reply:
added, and added the other methods that are missing these